src/models/pidforest.jl

```diff
@@ -9,17 +9,21 @@ mutable struct PIDForest
 		from pidforest.forest import Forest
 
 		def construct_pidforest(kwargs):
+			kwargs['sample_axis'] = 1
+			kwargs['threshold'] = 0
 			return Forest(**kwargs)
 		"""
 		new(py"construct_pidforest"(parameters))
 	end
 end
 
 function StatsBase.fit!(model::PIDForest, X::Array{T, 2}) where T<:Real
+	# no need for data transposition as the forest is learned on column major
     model.forest.fit(X)
 end
 
-function StatsBase.predict(model::PIDForest, x; err=0.1, pct=50) where T<:Real
-    -model.forest.predict(x; err=0.1, pct=50)[end] # last element contains scores
+function StatsBase.predict(model::PIDForest, x; pct=50) where T<:Real
+	# anomaly scores correspond to percentile `pct` over trees
+	# the lower the score the more anomalous the sample should be
```

Comment:
usually, this is the other way around, i.e. the higher the score, the more anomalous the sample is (think reconstruction error)

Reply:
I described the internals of pidforest here, where the lower the score the more anomalous the sample is. That's why I've put here the minus sign and that is also what the authors had done when computing the evaluation metrics.
https://github.com/janfrancu/pidforest/blob/master/scripts/run_classification_sklearn_pidforest_alg.py#L75-L82

Reply:
Ok, that makes sense, I was just trying to make sure we use the same convention